client/squawkers.js

```diff
@@ -75,10 +85,60 @@ class SquawkerItem extends React.Component {
       })).then(() => {
         this.audioEl.play();
         this.videoEl.play();
+        this.sendFileData(reliableChannel, unreliableChannel);
       });
     });
   }
 
+  sendFileData(reliableChannel, unreliableChannel) {
+    const dataFile = this.props.squawker.dataFile;
+    if (!dataFile) { return; }
+
+    const reader = new FileReader();
+    reader.onload = () => {
+      const messages = JSON.parse(reader.result);
+      const start = performance.now();
+      let index = 0;
+      const userId = this.props.squawker.userId;
+      const messageIntervalId = setInterval(() => {
+        const time = performance.now() - start;
+        let message = messages[index];
+        while (time >= message.time) {
+          if (message.message.data.owner) {
+            message.message.data.owner = userId;
+          }
+          if (message.message.data.networkId) {
+            message.message.data.networkId += userId;
+          }
+          if (message.message.data.parent) {
+            message.message.data.parent += userId;
+          }
+          message.message.clientId = userId;
```

Comment:
Hmm, just realized this data munging is specific to networked-aframe, so I'm kinda mixing concerns here. Maybe this can live in the naf-janus-adapter repo in the future.

Reply:
My guess is that the "right thing" is that NAF generates the recordings in a portable format, and then the playback code can live anywhere.

Reply:
Hmm, maybe. Anyway this is not meant to be a permanent recording solution, so I'm not going to stress about it now.